src/atomic.py

```diff
@@ -79,6 +79,10 @@ def look_at_point(x_node, y_node, z_node):
 
 def look_at_face_point(x, y, z):
 	evl.look_at_point(x, y, z)
+	#evl.gaze_at_point(x, y, z)
```

Comment:
It has been observed that evl.look_at_point changes the head direction as well as gaze, so I commented the gaze from here, i had added it previously not knowing that loo at point will also change gaze. As you mentioned I should probably delete stuff not required.